test/test-debuglet.js

```diff
@@ -120,6 +122,61 @@ describe('Debuglet', function() {
       debuglet.start();
     });
 
+    it('should use GCLOUD_PROJECT in lieu of config.projectId', function(done) {
+      process.env.GCLOUD_PROJECT = '11020304f2934-b';
```

Comment:
When setting this do you save the old value and then restore it? I can't seem to find it in this change. Only reason I advocate for this is this was a huge pain in growing errors tests where multiple test files ended up using and modifying the `env`